dashboard/src/features/shared/components/PageTitle/PageTitle.jsx

```diff
@@ -47,7 +47,7 @@ class PageTitle extends React.Component {
 }
 
 const mapStateToProps = (state) => {
-  const routes = makeRoutes()
+  const routes = makeRoutes().childRoutes.find(route => route.useForBreadcrumbs)
```

Comment:
`routes` feels like not quite the right name here, as `find` returns a singular thing.